### ISSUE TITLE
Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Right/left arrow keys and also with T key JAWS focus is not landing on table: A11y_Testing, Interactive and ARM tools_Table_Screen Reader. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/interactive#3596]
### GIT Hub Tags:
#AI Limited; #A11yTCS; #A11ySev2; #Testing, Interactive and ARM tools; #BM_Testing-Interactive and ARM tools_Win32_JULY2024; #DesktopApp; #JAWS; #WCAG1.3.1; #Screen Reader; #FTP; #A11yMAS; #Info and Relationship;

### Environment and OS details:
Windows 11 enterprise
OS build 22631.3737

### Reproduction Steps

1. Turn on JAWS and Install "Visual Studio Code" application.
2. Install .NET 6.0 SDK using https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.402-windows-x64-installer.
3. Install Polyglot Notebook extension. 
4. launch Visual studio code insider and Ctrl+Alt+Shift+N will invoke "Polyglot Notebook: Create new blank notebook" command used above.
5. Enter any code and run the code.
6. Tab till values and press enter.
7. In polyglotntebooks:values section tab till "table".
8. Observe that Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Right/left arrow keys and also with T key JAWS focus is not landing on table.

### Actual behavior:
Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Right/left arrow keys and also with T key JAWS focus is not landing on table.

### Expected behavior:
Jaws should be announcing table cells when user is using universal table keys Ctrl + Alt + Right/left arrow keys and also with T key JAWS focus should be landing on table.

### User Impact:
Visually impaired users who relay on screen reader will face difficulty if Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Right/left arrow keys and also with T key JAWS focus is not landing on table.

### Attachments:
[Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Rightleft arrow keys.webm](https://github.com/dotnet/interactive/assets/98728920/ff5104e7-5878-49c0-8c89-e8a5afc28932)

[Jaws is not announcing table cells when user is using universal table keys Ctrl + Alt + Rightleft arrow keys and also with T key JAWS focus is not landing on table.webm](https://github.com/dotnet/interactive/assets/98728920/9da077d5-e7ac-4dfe-b456-13bfc4444e2d)

**Comment by Yash14j** [comment]
GithubTag:#Rev:yaja;

**Comment by AbhitejJohn** [comment]
@colombod : Do you know if this is something we can fix in our code?
